node/middlewares/sitemap.ts

```diff
@@ -30,8 +36,8 @@ export const sitemap = async (ctx: Context) => {
   })
   if (ctx.url === '/sitemap.xml') {
```

Comment:
When I tried to avoid cache I noticed this check doesn't work if the request comes with a query string. Can you fix it?